tests/php/test-fieldmanager-context-post.php

```diff
@@ -133,6 +147,8 @@ public function test_context_save() {
 	}
 
 	public function test_programmatic_save_posts() {
+		$this->expectNotToPerformAssertions();
```

Comment:
This test doesn't do anything and should either be updated to perform assertions or removed entirely.